src/components/Author/styles.css

```diff
@@ -0,0 +1,82 @@
+.author_card {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.author_card .discord a {
+    color: var(--ifm-color-primary-darker);
+    font-weight: bold;
+}
+
+.author_card a {
+    flex-shrink: 0;
+}
+
+.author_card a img {
+    transition: transform 0.3s, opacity 0.3s;
+    width: 60px;
+    height: 60px;
+}
+
+.author_card a img:hover {
+    opacity: 0.9;
+    transform: scale(1.03);
+}
+
+.author_card span,
+.author_card a {
+    font-size: 16px;
+    line-height: 1.4;
+}
+
+.author_card>div {
+    /* display: flex;
+    gap: 1rem; */
```

Comment:
could we remove the commented-out code? 🙏

Reply:
of course. Since it was already there I didn't want to mess with the things you guys already had there but I'll delete the commented code

Reply:
awesome, appreciated :)